server/server/server.go

```diff
@@ -373,7 +373,16 @@ func (s *Server) startLocalModeServer() error {
 		ConnContext: api.SetContextConn,
 	}
 
-	// TODO: Close and delete socket file on shutdown
+	// If the socket file doesn't exist
```

Comment:
The comment is redundant to the code, and it is not helping to understand the intentions behind this. Why are we creating the file if it doesn't exist to just delete the file right after that?

Reply:
#### About your First Issue :
Yes, it is Redundant.
I kind of Prefer it with Little Comments to help me out faster.
But i have to Appreciate good Feedback and will therefore remove them :)

#### About the Second Part:
It gets Created because, if it doesn't Exist by Default, the Program throws an Error.
(Was part of the Issue because the (Linux) Path couldn't be found under Windows) 

About the Deletion: If you mean the Unlink, it is only deleted if no existing File Handlers are there to Point to the File. (E.g. the Socket is no longer Needed / The Program has Stopped)
 
I didnt find Proper Documentation in Go but i think under the Hood the same as in Linux is Achieved: [Linux Man Page Unlink](https://linux.die.net/man/2/unlink)

The File does get created, because it is Needed at Runtime for Communication between Programms. See [Wikipedia Socket File](https://en.wikipedia.org/wiki/Unix_file_types#:~:text=A%20socket%20is%20a%20special,and%20recvmsg()%20system%20calls.) For reference.

After the Process/Server Exits, the Socket File is no Longer Needed, that's why it should get deleted/cleaned up. (Apparently the Code is Redundant for Deletion, but it Still didn't get Deleted on My System)

I hope I wasn't too Over explanatory, but I wanted to make sure we are aware of the Reasoning from one another. 
Did this solve the Second Part of your Issue?
Or did I misunderstand your Problem?